packages/styles/src/icons/tk-icons.codepoints.json

```diff
@@ -91,8 +91,8 @@
     "minus-round": 61786,
     "minus": 61787,
     "mobile": 61788,
-    "more-": 61789,
-    "more": 61790,
+    "more-horizontal": 61789,
+    "more-vertical": 61790,
```

Comment:
I suggest to add an alias in file `tk-icons.aliases.json`, to not introduce regression for `more` icon.

```json
   "more": "more-vertical"
```

By doing this, the user can still use the `more` icon, which will use the `more-vertical` icon.
If you run `yarn` at the root level of UI-Toolkit, you will see the alias generations in the console (then don't forget to commit the tk-icons.codepoints.json file)

Reply:
Ah, perfect! Updating.